js/js.translator/testData/box/native/passExtLambdaFromNative.js

```diff
@@ -3,5 +3,5 @@ function A(v) {
 }
 
 function nativeBox(b) {
-    return b.bar_a13ma$(new A("foo"), function($reciever, i, s) { return "" + $reciever.v + s + i })
+    return b.bar_a13ma$(new A("foo"), function($receiver, i, s) { return "" + $reciever.v + s + i })
```

Comment:
Even though I don't know much about JS backend, I'm pretty sure that this change will break test